imagevector/images.yaml

```diff
@@ -380,8 +380,8 @@ images:
   sourceRepository: github.com/gardener/remedy-controller
   resourceId:
     name: remedy-controller-image
-  repository: europe-docker.pkg.dev/gardener-project/releases/gardener/remedy-controller/remedy-controller-azure
-  tag: "v0.11.0"
+  repository: europe-docker.pkg.dev/gardener-project/snapshots/gardener/remedy-controller-azure
+  tag: "v-dev-123" # TODO: fix this
```

Comment:
You can revert this change now in favor of https://github.com/gardener/gardener-extension-provider-azure/pull/1071 (or rebase once it is merged).

Reply:
Yes. I will rebase once it is merged

Reply:
All yours :)

Reply:
Done